### ISSUE TITLE
white highlights where no color

### ISSUE THREAD

**Issue by nagy135** [brenoprata10/nvim-highlight-colors#21]
Hello, I had to remove this plugin because it did this strange behavior to me but this is very likely nightly issue because it started happening after some update there.
I just felt like its good idea to report this, to prepare you, because this looks like something changed in neovim highlights ...feel free to close this.

I m running latest development branch of neovim **NVIM v0.8.0-dev-872-gbccb5de89d** ([neovim-git](https://aur.archlinux.org/packages/neovim-git) arch linux AUR package)

![image](https://user-images.githubusercontent.com/11056512/185070453-f47459d3-f713-43ee-b749-0fbe454e2674.png)

**Comment by brenoprata10** [comment]
Hello 👋
Which render option are you using?

**Comment by nagy135** [comment]
I am basically calling empty setup function. Everything is up to date, so whatever is default, thats what I m using. 
I will pull my dot files later today on another machine, maybe it will provide more insights.

**Comment by brenoprata10** [comment]
I don't think this is happening because of my plugin code, i think you should report it to the theme maintainer

**Comment by nagy135** [comment]
i m pretty sure this isnt your fault, it might be combination of some other plugin I have or some breaking change that happened in neovim nightly. I just opened issue so that the other guy facing this will have some starting point at least. 
And no it has nothing to do with color scheme, that was the first thing I ruled out. At this point there are countless plugins providing similair functionality so I just hopped, hopefully noone else will have to.
